addons/point_of_sale/static/tests/tours/utils/product_screen_util.js

```diff
@@ -279,14 +283,19 @@ export function clickFiscalPosition(name, checkIsNeeded = false) {
 
     if (checkIsNeeded) {
         step.push(
-            ...clickControlButtonMore(),
+            {
+                content: "click more button",
+                trigger: ".mobile-more-button",
+                mobile: true,
```

Comment:
```suggestion
                run: 'click'
                mobile: true,
```

There is a new development in tours. tour steps no longer have a default "run". If no "run" specified, it's just a check. https://github.com/odoo/odoo/pull/168474

Reply:
@caburj Changes are done. Can you please check again?
Thanks!